crates/sui-protocol-config/src/lib.rs

```diff
@@ -61,9 +61,10 @@ const MAX_PROTOCOL_VERSION: u64 = 20;
 // Version 19: Changes to sui-system package to enable liquid staking.
 //             Add limit for total size of events.
 //             Increase limit for number of events emitted to 1024.
-// Version 20: Enabling the flag `narwhal_new_leader_election_schedule` for the new narwhal leader
+// Version 20: Enables the flag `narwhal_new_leader_election_schedule` for the new narwhal leader
 //             schedule algorithm for enhanced fault tolerance and sets the bad node stake threshold
 //             value. Both values are set for all the environments except mainnet.
+//             Enables the flag `shared_object_deletion` allowing shared objects to be deleted.
```

Comment:
I don't see the flag enabled below for 20?

Reply:
yes, we only want the flag enabled with whatever version, likely 21, and above